src/patterns/help-users-to/correct-errors/examples/errors-proto/errors/index.njk

```diff
@@ -425,7 +423,7 @@ layout: none
                         "checked": true,
                         "value": "pepperoni",
                         "other": {
-                            "otherType": "select",
+                            "otherType": "input",
```

Comment:
This doesn't match the state of the previous page now. I don't think we even need this example though, to be honest. We're already showing an example of a nested select error above.